addons/point_of_sale/models/pos_order.py

```diff
@@ -141,7 +141,7 @@ def _process_order(self, order, draft, existing_order):
         if to_invoice:
             pos_order.action_pos_order_invoice()
             pos_order.account_move.sudo().with_context(force_company=self.env.user.company_id.id).post()
-
+        pos_order._create_order_picking()
```

Comment:
The picking of the invoiced order should be created first before calling `post` to the `pos_order.account_move`.

This is the reason why `test_pos_stock_account.py` fails at line 191.

`_get_pos_anglo_saxon_price_unit` is in the call stack of line 143 and it needs the `move_lines` in the `picking_ids`.